src/extras/gizmo/translate-gizmo.js

```diff
@@ -305,18 +311,31 @@ class TranslateGizmo extends TransformGizmo {
         return this._shapes.face.tolerance;
     }
 
+    /**
+     * @param {string} prop - The property to set.
+     * @param {any} value - The value to set.
```

Comment:
OK, I'm gonna stop pointing these out now....maybe there's a good reason it's `any`! 😄

Reply:
All of these are any because its a generic property update so the property could technically be any type